output/components.sasslint.txt

```diff
@@ -6,11 +6,18 @@
 src/Actions/ActionFile/ActionFile.scss
   5:12  error  Strings must use single quotes  quotes
 
+src/Status/Status.scss
+  10:1  warning  Space expected between blocks  empty-line-between-blocks
+  13:1  warning  Space expected between blocks  empty-line-between-blocks
+  18:1  warning  Space expected between blocks  empty-line-between-blocks
+  24:1  warning  Space expected between blocks  empty-line-between-blocks
+  28:1  warning  Space expected between blocks  empty-line-between-blocks
```

Comment:
WAT ?

Reply:
Oh, I will update `.sass-lint.yml` on the tools repository, thanks

Reply:
https://github.com/Talend/tools/pull/54 :)